docker/config/patch.diff

```diff
@@ -169,45 +170,51 @@ index 0000000..2c7fceb
 +
 +    def ingest(self, data, name, **kwargs):
 +        from geopyspark.geotrellis.rdd import RasterRDD, TiledRasterRDD
-+        from geopyspark.geotrellis.constants import ZOOM
++        from geopyspark.geotrellis.render import PngRDD
++        # from geopyspark.geotrellis.constants import ZOOM
 +
 +        rdd = data.rdd
 +        if isinstance(rdd, RasterRDD):
 +            metadata = rdd.collect_metadata()
 +            laid_out = rdd.tile_to_layout(metadata)
-+            reprojected = laid_out.reproject("EPSG:3857", scheme=ZOOM)
++            # reprojected = laid_out.reproject("EPSG:3857", scheme=ZOOM)
++            png = PngRDD.makePyramid(laid_out, data.rampname)
 +        elif isinstance(rdd, TiledRasterRDD):
 +            laid_out = rdd
-+            reprojected = laid_out.reproject("EPSG:3857", scheme=ZOOM)
++            # reprojected = laid_out.reproject("EPSG:3857", scheme=ZOOM)
++            png = PngRDD.makePyramid(laid_out, data.rampname)
++        elif isinstance(rdd, PngRDD):
++            png = rdd
 +        else:
 +            raise Exception
```

Comment:
This should raise a `TypeError` with a description of the error, not just a plain `Exception`.